### ISSUE TITLE
op16_no03: vhv does not render staff placement properly due to clef change around spine split

### ISSUE THREAD

**Issue by bel28kent** [bel28kent/Mysterium#41]
<img width="1440" alt="Screen Shot 2024-04-05 at 12 27 44 PM" src="https://github.com/bel28kent/Mysterium/assets/81043032/f43d9ff9-588b-441c-9ee8-bba30f1263fb">

**Comment by craigsapp** [comment]
Yes to the comment in another isusse: I currently require the clef change to be in the first subspine (leftmost) for the staff.  An easy solution is to this would be to include the clef change in all subspines for the staff:

<img width="971" alt="Screenshot 2024-04-10 at 6 31 35 PM" src="https://github.com/bel28kent/Mysterium/assets/3487289/94df2d5d-a4b0-461e-91a4-af5ab1b1225c">

[[View in VHV](https://verovio.humdrum.org?t=KiprZXJuCipjbGVmRzIKPTEKMWMKPTIKKl4KMmUJMmMKKgkqY2xlZkY0CjJHCTJCCip2CSp2Cj0xfHwKKmNsZWZHMgoxYwo9MgoqXgoyZQkyYwoqY2xlZkY0CSpjbGVmRjQKMkIJMkcKKnYJKnYKPT0KKi0K)]

I did the limitation since there was one visible clef, and I didn't want the confusing case where two different clefs were encoded in the subspines.

But It would be good to adjust the converter to allow for your cases using flipper.   Doing a clef change in all subspines is useful for extracting subspines:

<img width="971" alt="Screenshot 2024-04-10 at 6 35 39 PM" src="https://github.com/bel28kent/Mysterium/assets/3487289/8efadf21-a12a-4193-bc5d-f7b0d2ec4d19">

[[View in VHV](https://verovio.humdrum.org?t=KiprZXJuCipjbGVmRzIKPTEKMWMKPTIKKl4KMmUJMmMKKgkqY2xlZkY0CjJHCTJCCip2CSp2Cj0xfHwKKmNsZWZHMgoxYwo9MgoqXgoyZQkyYwoqY2xlZkY0CSpjbGVmRjQKMkIJMkcKKnYJKnYKPT0KKi0KISEhZmlsdGVyOiBleHRyYWN0eCAtcyAxYSwxYgo=)]

Here I separate the two subspines into separat staves.  In order to preserved clef changes in both staves, it has to be in both subspines (so that still might be the best way to encode, which allows viewing the voices on separate staves (or also useful for some types of analyses where you want to deal single-voice spines).

<img width="971" alt="Screenshot 2024-04-10 at 6 40 50 PM" src="https://github.com/bel28kent/Mysterium/assets/3487289/1140fe95-aae6-426a-93d4-1a14fdddef7d">

[[View in VHV](https://verovio.humdrum.org?t=KiprZXJuCSoqa2VybgoqY2xlZkcyCSpjbGVmRzIKPTEJPTEKMWMJMWMKPTIJPTIKKgkqCjJlCTJjCioJKmNsZWZGNAoyRwkyQgoqCSoKPTF8fAk9MXx8CipjbGVmRzIJKmNsZWZHMgoxYwkxYwo9Mgk9MgoqCSoKMmUJMmMKKmNsZWZGNAkqY2xlZkY0CjJCCTJHCioJKgo9PQk9PQoqLQkqLQohISFYZmlsdGVyOiBleHRyYWN0eCAtcyAxYSwxYgo=)]

**Comment by bel28kent** [comment]
> I did the limitation since there was one visible clef, and I didn't want the confusing case where two different clefs were encoded in the subspines.

It might be worth considering this case further. For example, in [scriabin-op08_no12](https://github.com/bel28kent/Mysterium/blob/22-phase21-subspine-issue/op08/scriabin-op08_no12.krn), I'm using hidden rests to avoid excessive merging and splitting (I would have to merge and split twice in every measure otherwise). In m. 40, the left hand moves to treble clef, but only in the right subspine. Ideally I would add `*clefF4` in the left subspine to make this clear.

<img width="1440" alt="Screen Shot 2024-04-11 at 5 44 40 PM" src="https://github.com/bel28kent/Mysterium/assets/81043032/9a2e6b8d-bd93-4920-83cc-f8db337264df">

If I add the `*clefF4` the notes are correct, but both clefs are rendered on top of each other.

<img width="1440" alt="Screen Shot 2024-04-11 at 5 46 10 PM" src="https://github.com/bel28kent/Mysterium/assets/81043032/54ea4444-8dee-43de-8370-874832900b62">
